library/ssl_tls13_server.c

```diff
@@ -715,6 +715,9 @@ int mbedtls_ssl_parse_client_psk_identity_ext(
                         }
                         else
                         {
+                            MBEDTLS_SSL_DEBUG_MSG( 3,
+                            ( "0-RTT is disabled ( diff=%d exeeds MBEDTLS_SSL_EARLY_DATA_MAX_DELAY )",
```

Comment:
* `s/exeeds/exceeds`
* Shouldn't this be `MBEDTLS_SSL_TICKET_AGE_TOLERANCE`?
* Overly long line.

Reply:
I believe it is `MBEDTLS_SSL_EARLY_DATA_MAX_DELAY`, the condition is checked on L711.

Reply:
My bad!